aws-spot-instances.md

```diff
@@ -23,6 +31,69 @@ Spot data feeds are an account level setting, not a payer level. Every AWS Accou
 For Spot data written to an S3 bucket only accessed by Kubecost, it is safe to delete objects after three days of retention.
 {% endhint %}
 
+## Configuring IAM
+
+Kubecost requires read access to the Spot data feed bucket. The following IAM policy can be used to grant Kubecost read access to the Spot data feed bucket.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+        "Sid": "SpotDataFeed",
+        "Effect": "Allow",
+        "Action": [
+          "s3:ListAllMyBuckets",
+          "s3:ListBucket",
+          "s3:HeadBucket",
+          "s3:HeadObject",
+          "s3:List*",
+          "s3:Get*"
+        ],
+        "Resource": "arn:aws:s3:::${SpotDataFeedBucketName}*"
+    }
+  ]
+}
+```
+
+To attach the IAM policy to the Kubecost service account, you can use IRSA or the account's service key.
+
+### Option 1: IRSA (IAM Roles for Service Accounts)
+
+```bash
+eksctl create iamserviceaccount \
```

Comment:
Will this overwrite an existing service account annotation?

Reply:
Hmm good catch, I think it probably will. Would it help to add the following note?

> If your `serviceaccount/kubecost-cost-analyzer` already has IRSA annotations attached, be sure to include all policies necessary when running this command.

Reply:
That works!